plugins/proxy-backend/README.md

```diff
@@ -19,8 +19,8 @@ const proxyEnv = useHotMemoize(module, () => createEnv('proxy'));
 
 const service = createServiceBuilder(module)
```

Comment:
These docs are too hidden. Move this to https://backstage.io/docs/plugins/proxying

Reply:
done!